ddl/ddl.go

```diff
@@ -90,17 +99,20 @@ func (d *ddl) CreateSchema(ctx context.Context, schema model.CIStr) (err error)
 		return errors.Trace(ErrExists)
 	}
 	info := &model.DBInfo{Name: schema}
-	info.ID, err = meta.GenGlobalID(d.store)
+	info.ID, err = d.meta.GenGlobalID()
 	if err != nil {
 		return errors.Trace(err)
 	}
 
-	err = kv.RunInNewTxn(d.store, false, func(txn kv.Transaction) error {
-		err := d.verifySchemaMetaVersion(txn, is.SchemaMetaVersion())
+	err = d.meta.RunInNewTxn(false, func(m *meta.TMeta) error {
+		err := d.verifySchemaMetaVersion(m, is.SchemaMetaVersion())
 		if err != nil {
 			return errors.Trace(err)
 		}
-		err = d.writeSchemaInfo(info, txn)
+
+		err = m.CreateDatabase(info)
+
+		log.Warnf("save schema %s", info)
```

Comment:
This is the old log, just keep it here now.
I will handle whole in following async schema changes.